React/Base/RCTDisplayLink.m

```diff
@@ -96,6 +96,13 @@ - (void)dealloc
 
 - (void)invalidate
 {
+  // ensure observer callbacks do not hold a reference to weak self via pauseCallback
```

Comment:
Any changes to files that exist upstream need to be in a TODO tag ([example in this PR](https://github.com/microsoft/react-native-macos/pull/857/files)). It makes merges way easier when we know what diffs are purposeful or not